### PR TITLE
snip bad dependencies from openshift-controller-manager

### PR DESCRIPTION
**Pull request by deads2k**
/assign @damemi

### CODE REVIEW

File: pkg/route/apiserver/registry/route/etcd/etcd.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package etcd
 import (
 	"context"
 
+	"github.com/openshift/origin/pkg/route/apiserver/routeinterfaces"
```
Comment (damemi) [inline pkg/route/apiserver/registry/route/etcd/etcd.go]: should this be with the other openshift imports?

File: pkg/route/apiserver/registry/route/etcd/etcd_test.go [hunk 2]
```diff
@@ -3,6 +3,8 @@ package etcd
 import (
 	"testing"
 
+	"github.com/openshift/origin/pkg/route/apiserver/routeinterfaces"
```
Comment (damemi) [inline pkg/route/apiserver/registry/route/etcd/etcd_test.go]: same, import order

File: pkg/route/apiserver/registry/route/strategy.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/openshift/origin/pkg/route/apiserver/routeinterfaces"
```
Comment (damemi) [inline pkg/route/apiserver/registry/route/strategy.go]: import order

File: pkg/route/apiserver/routeallocationcontroller/route_allocation_controller.go [hunk 4]
```diff
@@ -1,16 +1,16 @@
-package allocation
+package routeallocationcontroller
 
 import (
+	"github.com/openshift/origin/pkg/route/apiserver/routeinterfaces"
```
Comment (damemi) [inline pkg/route/apiserver/routeallocationcontroller/route_allocation_controller.go]: import order

File: pkg/build/apiserver/admission/strategyrestrictions/admission.go [hunk 5]
```diff
@@ -5,6 +5,8 @@ import (
 	"io"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/runtime"
```
Comment (damemi) [inline pkg/build/apiserver/admission/strategyrestrictions/admission.go]: import order

File: pkg/image/importer/importer.go [hunk 6]
```diff
@@ -6,6 +6,8 @@ import (
 	"runtime"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/runtime/schema"
```
Comment (damemi) [inline pkg/image/importer/importer.go]: import order

**Comment by damemi** [comment]
looks good if you fix those weird import orders

**Comment by deads2k** [comment]
replaced by https://github.com/openshift/origin/pull/23012

**Pull request closed without merging**